charts/rancher-backup/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: rancher/backup-restore-operator
-  tag: v1.0.3
+  tag: latest
```

Comment:
Why do we need to change this tag?  I noticed its real value will be determined by the build scripts. 

`rancher/backup-restore-operator:latest` does not exist in dockerhub.

Reply:
The thought was that, like `rancher-operator`, we would use a development tag in-tree. This gets overwritten by `scripts/package-helm` anyway, like you said.

However, you are correct that `latest` does not exist, so I will revert this change for this PR. We will most likely need to ensure that it gets changed in the future.